### PR TITLE
Updated Contribution Guidelines.

### PR DESCRIPTION
**Pull request by saulotoledo**
A suggestion to reinforce the contribution guidelines to avoid that new contributors make the same mistake I made.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Also, please ensure your pull request adheres to the following guidelines:
 * Software that is PHP 5.5+, Composer-installable, PSR compliant, semantically versioned, united tested and well documented in English is preferred.
 * Please search previous suggestions before making a new one, as yours may be a duplicate.
 * Enter a meaningful pull request description.
-* Please make an individual commit for each suggestion in a pull request.
+* Please make an individual commit for each suggestion in a separated pull request.
```
Comment (jadb) [inline CONTRIBUTING.md]: Maybe use `separate` instead of `separated`?

**Comment by saulotoledo** [comment]
@jadb Thank you. It is done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (saulotoledo) [inline CONTRIBUTING.md]: @jadb Thank you. It is done.

**Comment by josegonzalez** [comment]
Thanks!

**Pull request merged**